SC_redesign/Crop_and_Soil/soil/carbon_cycling/pool_gas_partition.py

```diff
@@ -0,0 +1,629 @@
+        """
+        return plant_metabolic_active_carbon_remaining + plant_structural_active_carbon_remaining
+
+    @staticmethod
+    def _soil_active_decompose_carbon(soil_metabolic_active_carbon_remaining: float,
+                                      soil_structural_active_carbon_remaining: float) -> float:
+        """
+        Calculates soil carbon decomposed into the active carbon pool (kg/ha)
+        Args:
+            soil_metabolic_active_carbon_remaining:
+                    soil metabolic carbon decomposed to active carbon after accounting for carbon dioxide loss (kg/ha)
+            soil_structural_active_carbon_remaining:
+                    soil structural carbon decomposed to active carbon after accounting for carbon dioxide loss (kg/ha)
+
+        Returns: Soil carbon decomposed into the active carbon pool (kg/ha)
+
+        pseudocode_soil Reference: S.6.C.11
+        """
+        return soil_metabolic_active_carbon_remaining + soil_structural_active_carbon_remaining
+
+    @staticmethod
+    def _soil_active_carbon_amount(active_carbon_amount: float, plant_active_decompose_carbon: float,
+                                   soil_active_decompose_carbon: float, passive_to_active_carbon_amount: float,
+                                   slow_to_active_carbon_amount: float,
+                                   active_carbon_decomposition_amount: float) -> float:
+        """
+        Aggregate the total amount of active carbon in the layer
+
+        Args:
+            active_carbon_amount: active carbon stored in the soil (kg/ha)
+            plant_active_decompose_carbon: plant carbon decomposed into the active carbon pool (kg/ha)
+            soil_active_decompose_carbon: soil carbon decomposed into the active carbon pool (kg/ha)
+            passive_to_active_carbon_amount: passive carbon decomposed into active carbon (kg/ha)
+            slow_to_active_carbon_amount: slow carbon decomposed into active carbon (kg/ha)
+            active_carbon_decomposition_amount: active carbon decomposed into slow or passive carbon and CO2 (kg/ha)
+
+        Returns: Updated active carbon stored in the soil (kg/ha)
+
+        pseudocode_soil Reference: S.6.C.11
+        """
+        return active_carbon_amount + plant_active_decompose_carbon + soil_active_decompose_carbon \
+               + slow_to_active_carbon_amount + passive_to_active_carbon_amount - active_carbon_decomposition_amount
+
+    @staticmethod
+    def _passive_to_active_carbon_amount(passive_carbon_decomposition_amount: float,
+                                         passive_carbon_co2_lost_rate=0.55) -> float:
+        """
+        Calculates passive carbon decomposed into active carbon (kg/ha)
+
+        Args:
+            passive_carbon_decomposition_amount: passive carbon decomposed into active or passive carbon and CO2 (kg/ha)
+            passive_carbon_co2_lost_rate: passive carbon lost as CO2 during decomposition (%)
```

Comment:
I've suggested a change to this variable. Carbon and CO2 are redundant and this is the "loss rate" not "lost rate":

```suggestion
            passive_carbon_loss_rate: passive carbon lost as CO2 during decomposition (%)
```